src/toggle-switch/toggle-switch.scss

```diff
@@ -0,0 +1,235 @@
+.ToggleSwitch {
+  align-items: center;
+  display: inline-flex;
+
+  &:hover {
+    .ToggleSwitch-knob {
+      background-color: var(--color-btn-hover-bg);
+    }
+  }
+
+  &:active {
+    .ToggleSwitch-knob {
+      background-color: var(--color-btn-active-bg);
+    }
+  }
+}
+
+.ToggleSwitch-track {
+  position: relative;
+  display: block;
+  width: 64px;
+  height: 32px;
+  padding: 0;
+  text-decoration: none;
+  cursor: pointer;
+  user-select: none;
+  background-color: var(--color-switch-track-bg);
+  border-color: var(--color-switch-track-border);
+  border-style: $border-style;
+  border-width: $border-width;
+  border-radius: $border-radius;
+  outline-offset: 2px;
+  transition-timing-function: cubic-bezier(0.5, 1, 0.89, 1);
+  transition-duration: 80ms;
+  transition-property: background-color, border-color;
+  appearance: none;
+
+  @media (pointer: coarse) {
+    &::before {
+      position: absolute;
+      top: 50%;
+      right: 0;
+      left: 0;
+      min-height: 44px;
+      content: '';
+      transform: translateY(-50%);
+    }
+  }
+
+  @media (prefers-reduced-motion) {
+    transition: none;
+
+    * {
+      transition: none;
+    }
+  }
+
+  &::after {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    box-sizing: border-box;
+    content: '';
+    border-radius: $border-radius;
+  }
```

Comment:
We don't need an `::after` anymore. I forgot to remove it when I changed styles in the Primer React PR.

```suggestion
```